CODEOWNERS

```diff
@@ -4,8 +4,8 @@
 
 # release configuration
 
-/.release/                               @hashicorp/github-secure-boundary
-/.github/workflows/build.yml             @hashicorp/github-secure-boundary
```

Comment:
Hm, I was using https://github.com/orgs/hashicorp/teams/team-selfmanaged-releng, but it appears this doesn't have access to the repo? I'll ping someone on the releng team for clarification.